seccomp/README.md

```diff
@@ -185,47 +348,37 @@ When writing a seccomp filter sometimes arguments are truncated by the operating
-If you need access to devices use `--device`.
+* Using the `--privileged` flag when creating a container disables seccomp in all versions of docker - even if you explicitly specify a seccomp profile. In general you should avoid using the `--privileged` flag as it does too many things. You can achieve the same goal with `--cap-add ALL --security-opt apparmor=unconfined --security-opt seccomp=unconfined`. If you need access to devices use `--device`.
 
-* In docker 1.10-1.12+ docker exec `--privileged` does not bypass seccomp. This may change in future versions https://github.com/docker/docker/issues/21984
+* In docker 1.10-1.12 `docker exec --privileged` does not bypass seccomp. This may change in future versions https://github.com/docker/docker/issues/21984.
 
-* In docker 1.12+ adding a capability disables the relevant seccomp filter in the default seccomp profile. It can't disable apparmor, though.
+* In docker 1.12 and later, adding a capability disables the relevant seccomp filter in the default seccomp profile. However, it does not disable apparmor.
 
 ### Using multiple filters
 
-The only way to use multiple seccomp filters right now is to load addition filters within your program at run time.
-The kernel supports layering filters.
-When using multiple layered filters they are always all executed, starting with the most recently added.
-The highest precedence action returned is taken.
-See the man page for all the details: http://man7.org/linux/man-pages/man2/seccomp.2.html
+The only way to use multiple seccomp filters, as of Docker 1.12, is to load additional filters within your program at runtime. The kernel supports layering filters.
+
+When using multiple layered filters, all filters are always executed starting with the most recently added. The highest precedence action returned is taken. See the man page for all the details: http://man7.org/linux/man-pages/man2/seccomp.2.html
 
 ### Misc
 
-You can enable JITing of BPF filters (if it isn't already enabled) this way:
+You can enable JITing of BPF filters (if it isn't already enabled) with the following command:
 
 ```
 $ echo 1 > /proc/sys/net/core/bpf_jit_enable
 ```
 
-There is no easy to use seccomp in a mode that reports errors without crashing the program, but it's possible to implement in one of several ways.
-One way to do this is to use SCMP_ACT_TRAP and make your process handle SIGSYS and report the error in a useful way.
-There's some information about how Firefox handles seccomp violations here: https://wiki.mozilla.org/Security/Sandbox/Seccomp
+There is no easy way to use seccomp in a mode that reports errors without crashing the program. However, there are several round-about ways to accomplish this. One such way is to use SCMP_ACT_TRAP and write your code to handle SIGSYS and report the errors in a useful way. Here is some information on [how Firefox handles seccomp violations](https://wiki.mozilla.org/Security/Sandbox/Seccomp).
 
```

Comment:
Some users have reported seccomp errors appearing in audit logs, I think only with ACT_KILL. I need to double check this on a system with auditd eg redhat, as it is useful.